pom.xml

```diff
@@ -37,7 +37,7 @@
     <dependency>
       <groupId>com.amazonaws</groupId>
       <artifactId>aws-java-sdk</artifactId>
-      <version>1.10.61</version>
+      <version>1.11.18</version>
```

Comment:
Upgrade & Builder change both of the part of the PR. Shouldn't the title say so?